spring-cloud-gcp-secretmanager/src/main/java/org/springframework/cloud/gcp/secretmanager/SecretManagerOperations.java

```diff
@@ -97,10 +97,10 @@
 	 * Enables the secret version under the specified project.
 	 *
 	 * @param secretId the secret ID of the secret to delete.
```

Comment:
Typo here. Should be "secret ID of the secret to enable"

Reply:
Nice catch, done.